pkg/store/builtin/build_sql.go

```diff
@@ -596,6 +596,79 @@ WHERE build_repo_id = ?
 AND build_number = ?
 `
 
+const stmtBuildSelectPullRequestNumber = `
+SELECT
```

Comment:
since the build_sql.go is auto-generated, let's move these non-generated queries to `build.go` so they don't overwritten. I haven't published the sql generator yet, but I'll try to do that next week :)

also, I will probably adjust some of the non-generated queries I wrote to use the following:

```Go
const stmtBuildSelectPullRequestNumber = stmtBuildSelectList + `
WHERE build_repo_id = ?
AND build_pull_request_number = ?
ORDER BY build_number DESC
LIMIT 1
`
```

... the benefit to the above is that we don't have to worry about column changes in our non-generated queries